CONTRIBUTING.md

```diff
@@ -12,16 +14,18 @@ In order to create a pull request for DAO Job Board, follow the GitHub instructi
 
 ## Installation
 
-1. Clone this repository and install dependencies.
+1. [Install Yarn](https://classic.yarnpkg.com/lang/en/docs/install) if you haven't already.
+2. Clone this repository and install dependencies.
+
    ```bash
    git clone https://github.com/Developer-DAO/DAO-job-board
 
    npm install # or `yarn install`
    ```
 
-2. Ask an invite to the Doppler Developer DAO workspace from [carlomigueldy.eth](https://github.com/carlomigueldy) or [angelgomez.eth](https://github.com/angeljgomezc)
-   
-3. Install Doppler CLI into your local machine
+3. Ask an invite to the Doppler Developer DAO workspace from [carlomigueldy.eth](https://github.com/carlomigueldy) or [angelgomez.eth](https://github.com/angeljgomezc)
```

Comment:
I can't attest to GH markdown, but one of the beauty of the numbered lists is that you can just put `1.` for each item and it will auto-increment. This eliminates the need to update them if you insert / delete an item from the list 💪

Reply:
Yes true I agree! The extension for VSCode for the markdown works pretty well, it's very convenient that it auto-increments